action.yml

```diff
@@ -109,6 +116,12 @@ runs:
     - name: Docker Build
       shell: bash
       run: ${{ github.action_path}}/scripts/docker_build
+    
+    - name: Run Test
+      shell: bash
+      run: ${{ inputs.smoke_test }}
+      env:
+        IMAGE_NAME: ${{env.TARBALL_NAME}}
```

Comment:
Could be a lot of work to pass to our users but I'd love to hear about other's opinion on this. Might be worthwhile to have a group discussion.

Reply:
Also, all Docker images right now are `linux/something` so maybe the cross-platform thing for architectures isn't such a big deal, since all linuxes are going to be able to run a bash script as the test...

Reply:
I just posted in our team room in Slack about this to talk through it with myself.. I think I'm overcomplicating things, we don't need to worry about non-linux platforms for now, and we can probably avoid needing to specify architecture because shell scripts themselves are cross-platform (within linux platforms), as long as they don't call any platform-specific binaries. Sorry for the noise @jeanneryan, thanks for bearing with me on this.

Reply:
Outcome of the Slack back & forth: there's a case for supporting both the simple script param as well as an output with the multi-arch manifest tag. I'm not 100% sold on doing both at once as adding the output will be more costly than it sounds in terms of story points. I am 100% that we will want the simple param though, so let's stick to that and add a new ticket if we later want to add the output as well.